template/haskell-http/src/Server.hs

```diff
@@ -45,5 +46,5 @@ getEnvironment varname defvalue = (varname,) . fromMaybe defvalue <$> lookupEnv
 envList :: [(String, String)]
 envList = [("HOSTNAME", "localhost"), ("TEST","DEFAULT")]
```

Comment:
Is `envList` used now that you removed `getContext`? If not, please remove

Reply:
also... you may want to enable all Errors and Warnings so the compiler warns you about unused functions
http://dev.stephendiehl.com/hask/#cabal